tests/run-make/split-debuginfo/Makefile

```diff
@@ -30,342 +30,339 @@ ifeq ($(UNAME),Darwin)
 		$(RUSTC) foo.rs -g -C split-debuginfo=unpacked
 		ls $(TMPDIR)/*.o
 		[ ! -d $(TMPDIR)/foo.dSYM ]
-else
-	ifdef IS_WINDOWS
-		# Windows only supports packed debuginfo - nothing to test.
-		off:
-		packed:
-		unpacked:
+else ifdef IS_WINDOWS
+	# Windows only supports packed debuginfo - nothing to test.
+	off:
+	packed:
+	unpacked:
```

Comment:
Obviously, this won't work with GNU Make 3.80, which I suspect is what the CIs are probably using.